### PR TITLE
GLT-3264: fixed requesting bulk page with many items

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java [hunk 1]
```diff
@@ -164,4 +165,60 @@ private static boolean checkFile(File path) throws IOException {
...
+   * @param formData
+   * @param required
+   * @return the value mapped to the provided key if it is set and not empty; otherwise null
+   * @throws ClientErrorException if the value is required and missing or empty, or if the value is set and non-empty, but is not a valid
+   *           Integer
+   */
+  public static Integer getIntegerInput(String key, Map<String, String> formData, boolean required) {
+    String stringValue = getStringInput(key, formData, required);
+    try {
+      return stringValue == null ? null : new Integer(stringValue);
```
Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java:218]: This constructor is deprecated in newer versions of Java: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Integer.html#%3Cinit%3E(java.lang.String)

**Comment by djcooke** [comment]
@avarsava: fixed. I'd moved a function and missed updating a couple references, so there was a Javascript error

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java (hunk 1 above)
Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java]: changed to `Integer#parseInt`

Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java]: Changed to `Integer#valueOf`

**Pull request merged**